client/gefyra/__main__.py

```diff
@@ -24,12 +24,19 @@
 
 up_parser = action.add_parser("up")
 up_parser.add_argument(
-    "-e",
-    "--endpoint",
```

Comment:
Another idea. We could keep this parameter for a little with a deprecation warning.

Reply:
@Schille I added the parameter again - just to print out a warning and exit. Otherwise we would have to reimplement the whole splitting again.